pkg/apic/apiserver/models/api/v1/resourceinstance.go

```diff
@@ -118,3 +118,17 @@ func (ri *ResourceInstance) FromInstance(from *ResourceInstance) error {
 func (ri *ResourceInstance) GetRawResource() json.RawMessage {
 	return ri.rawResource
 }
+
+// GetRawResource gets the resource as bytes
+func (ri *ResourceInstance) GetAPIPath() string {
```

Comment:
Should we also check for the selfLink, or do you think it's unnecessary? Maybe call this function `GetSelfLink`

Reply:
Good call, I will actually at it to ResourceMeta, so no need to convert to an instance